locales/en/yourloops.json

```diff
@@ -84,6 +85,8 @@
   "cookies": "Cookies",
   "current-password": "Current password",
   "date-picker-header-date-format": "MMMM YYYY",
+  "dropdown-error-profession": "Please choose a profession",
```

Comment:
If possible (not always the case), because of Crowdin shortcoming & the cost related to translate to a new language (based on the number of words), avoid duplicate translations string.
I wish it won't be the case, but it's not in our hands.
> "dropdown-error-profession"
> "profession-dialog-tilte"